390/Olivia_Handoko.Rmd

```diff
@@ -0,0 +1,320 @@
+
+#summarise(group_by(new_scbi_stem1_alive,sp),length(sp))
+
+new_scbi_stem2_notalive <- scbi_stem2_notalive %>% 
+    filter(sp %in% c("libe")) %>%
+    filter (!dbh == 0)
+ #   filter(StemTag %in% c(1,2)) %>%
+
+
+
+new_scbi_stem2_alive <-scbi_stem2_alive %>%
+  filter(sp %in% c( "libe")) %>%
+ # filter(StemTag %in% c(1,2)) %>%
+  filter(quadrat <=300 | quadrat >= 1000) %>%
+  filter(treeID <= 300)
+
+
+new_scbi_stem2_alive <-new_scbi_stem2_alive[duplicated(new_scbi_stem2_alive$tag), ]
+new_scbi_stem2_alive <-new_scbi_stem2_alive[duplicated(new_scbi_stem2_alive$treeID), ]
+
+new_scbi_stem2_alive <- new_scbi_stem2_alive[order(as.Date(new_scbi_stem2_alive$ExactDate, format="%m/%d/%Y")),]
+
+new_scbi_stem3_notalive <- scbi_stem3_notalive %>% 
+    filter(sp %in% c("libe")) %>%
+  filter(DFstatus != "alive" & DFstatus != "prior" & DFstatus != "missing" & dbh != "NULL") 
+
+ # filter(StemTag %in% c(1,2)) %>%
+
+
+new_scbi_stem3_alive <-scbi_stem3_alive %>%
+  filter(sp %in% c( "libe")) %>%
+  #filter(StemTag %in% c(1,2)) %>%
+  filter(quadrat <=300 | quadrat >= 1000) %>%
+  filter(treeID <= 300)
+
+#new_scbi_stem3_alive <-new_scbi_stem3_alive[!duplicated(new_scbi_stem3_alive$tag), ]
+```
+
+```{r}
+#bind the alive and not alive data together 
+##
+total_alive <- rbind(new_scbi_stem1_alive, new_scbi_stem2_alivex) %>% select("sp", "quadrat", "dbh", "ExactDate","date","quadrat", "treeID", "StemTag", "tag", "MeasureID") %>%
+  filter(!dbh %in% c(0, "NULL"))  %>%
+  filter(StemTag != 10) %>%
+  group_by(treeID, StemTag) %>%
+  filter(n() >= 2 ) 
+#I want tree ID and StemTag frequency to be more than just one for ST and 3 for treeID
+
+   #filter(!treeID %in% c(3,13,14,15,18,35,40,42,53,56,58,67,68,87,105,110,189,191,200,201,209,213,214,219,222,230,234,236,251,253,258,259,260,262,269,272,274,276,280,281,282,287,288,289,290,297))
+
+as.data.frame((table(total_alive$treeID))) 
+as.data.frame((table(total_alive$StemTag)))
+
+total_alive %>% group_by(treeID, StemTag) %>% summarize(freq = n()) %>%
+  filter(freq >= 2)
+
+# total_alive <-total_alive[duplicated(treeID$StemTag), ] %>%
+#   filter(tag != 20001)
+
+total_notalive <- rbind(new_scbi_stem2_notalive, new_scbi_stem1_notalive, new_scbi_stem3_notalive) %>% select("sp", "dbh", "ExactDate", "quadrat", "date", "treeID", "StemTag", "tag", "MeasureID") %>%
+  filter(!dbh %in% c(0, "NULL")) 
+
+```
+
+```{r}
+ggplot(total_alive) + 
+  geom_line(aes(x=date, y=dbh, group = interaction(treeID, StemTag), color = quadrat)) + facet_wrap(~quadrat)
+
+
+
+#+
+  #geom_text(data = subset(new_scbi_stem1_notalive, ExactDate== "12/18/2008"), aes(label = quadrat, x = ExactDate, y = dbh))
+
+
+```
+```{r}
+ggplot(new_scbi_stem1_notalive) + 
+  geom_point(aes(x=date, y=dbh, group = treeID, color = quadrat))
+```
+```{r}
+ggplot(new_scbi_stem2_notalive) + 
+  geom_point(aes(x=date, y=dbh, group = treeID, color = quadrat))
+```
+
+
+```{r}
+ggplot(new_scbi_stem3_notalive) + 
```

Comment:
Try creating a `date` variable with 
```
library(lubridate)
new_scbi_stem3_notalive %>%
   mutate(date = mdy(ExactDate)
```
somewhere before. you'll get prettier date axes that way